### PR TITLE
Pull from Upstream

### PR DESCRIPTION
**Pull request by isantop**
This pull includes fixes for not having quotes around paths in the kernel and initrd update hooks.

TESTING:

Check the output of `uname -r`

Install the new version, then install a different kernel version and reboot. 

Check the output of `uname -r` again. It should match the kernel version that was just installed.

### CODE REVIEW

File: data/kernel/zz-kernelstub [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 INITRD="/boot/initrd.img-$1"
 
-kernelstub --verbose --kernel-path $2 --initrd-path $INITRD
+kernelstub --verbose --kernel-path "$2" --initrd-path "$INITRD"
```
Comment (jackpot51) [inline data/kernel/zz-kernelstub]: 😄

File: data/initramfs/zz-kernelstub [hunk 2]
```diff
@@ -2,4 +2,4 @@
 
 KERNEL="/boot/vmlinuz-$1"
 
-kernelstub --verbose --initrd-path $2 --kernel-path $KERNEL
+kernelstub --verbose --initrd-path "$2" --kernel-path "$KERNEL"
```
Comment (jackpot51) [inline data/initramfs/zz-kernelstub]: 😄

File: bin/kernelstub [hunk 3]
```diff
@@ -19,6 +18,9 @@ OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER
 TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF
 THIS SOFTWARE.
 
+Please see the provided LICENCE.txt file for additional distribution/copyright
```
Comment (jackpot51) [inline bin/kernelstub]: LICENSE is misspelled

File: kernelstub/application.py [hunk 4]
```diff
@@ -19,6 +18,9 @@
 TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF
 THIS SOFTWARE.
 
+Please see the provided LICENCE.txt file for additional distribution/copyright
```
Comment (jackpot51) [inline kernelstub/application.py]: LICENSE is misspelled

File: kernelstub/config.py [hunk 5]
```diff
@@ -18,6 +17,9 @@
 OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER
 TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF
 THIS SOFTWARE.
+
+Please see the provided LICENCE.txt file for additional distribution/copyright
```
Comment (jackpot51) [inline kernelstub/config.py]: LICENSE is misspelled

File: kernelstub/drive.py [hunk 6]
```diff
@@ -18,6 +17,9 @@
 OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER
 TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF
 THIS SOFTWARE.
+
+Please see the provided LICENCE.txt file for additional distribution/copyright
```
Comment (jackpot51) [inline kernelstub/drive.py]: LICENSE is misspelled

File: kernelstub/installer.py [hunk 7]
```diff
@@ -18,6 +17,9 @@
 OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER
 TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF
 THIS SOFTWARE.
+
+Please see the provided LICENCE.txt file for additional distribution/copyright
```
Comment (jackpot51) [inline kernelstub/installer.py]: LICENSE is misspelled

File: kernelstub/nvram.py [hunk 8]
```diff
@@ -18,6 +17,9 @@
 OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER
 TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF
 THIS SOFTWARE.
+
+Please see the provided LICENCE.txt file for additional distribution/copyright
```
Comment (jackpot51) [inline kernelstub/nvram.py]: LICENSE is misspelled

File: kernelstub/opsys.py [hunk 9]
```diff
@@ -18,6 +17,9 @@
 OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER
 TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF
 THIS SOFTWARE.
+
+Please see the provided LICENCE.txt file for additional distribution/copyright
```
Comment (jackpot51) [inline kernelstub/opsys.py]: LICENSE is misspelled

**Comment by jackpot51** [comment]
Also, the name of LICENSE.txt is misspelled.

**Comment by isantop** [comment]
Licence is a perfectly acceptable spelling of the word throughout most of the English speaking world.

File: bin/kernelstub (hunk 3 above)
Comment (brs17) [reply bin/kernelstub]: In the U.S.A, it should be spelled "LICENSE.txt".

Let's make that change and then we should be good to merge.

File: kernelstub/application.py (hunk 4 above)
Comment (brs17) [reply kernelstub/application.py]: In the U.S.A, it should be spelled "LICENSE.txt".

Let's make that change and then we should be good to merge.

File: kernelstub/config.py (hunk 5 above)
Comment (brs17) [reply kernelstub/config.py]: In the U.S.A, it should be spelled "LICENSE.txt".

Let's make that change and then we should be good to merge.

File: kernelstub/drive.py (hunk 6 above)
Comment (brs17) [reply kernelstub/drive.py]: In the U.S.A, it should be spelled "LICENSE.txt".

Let's make that change and then we should be good to merge.

File: kernelstub/installer.py (hunk 7 above)
Comment (brs17) [reply kernelstub/installer.py]: In the U.S.A, it should be spelled "LICENSE.txt".

Let's make that change and then we should be good to merge.

File: kernelstub/nvram.py (hunk 8 above)
Comment (brs17) [reply kernelstub/nvram.py]: In the U.S.A, it should be spelled "LICENSE.txt".

Let's make that change and then we should be good to merge.

File: kernelstub/opsys.py (hunk 9 above)
Comment (brs17) [reply kernelstub/opsys.py]: In the U.S.A, it should be spelled "LICENSE.txt".

Let's make that change and then we should be good to merge.

**Comment by isantop** [comment]
Licence is a perfectly acceptable spelling of the word throughout most of the English speaking world.

**Pull request closed without merging**